docs/part3.md

```diff
@@ -1 +1,178 @@
 # Part III: Numerical precision in _ab initio_ calculations.
+
+In this part you will learn about several new quantities and tools for working with numerical convergence in NOMAD.
+These tools will help you
+
+1. query for the level of precision or conformity that you need.
+2. deploy extracted data into a notebook to start performing data science.
+
+Your starting point is the Entries page > FILTERS (side menu) > Precision.
+To navigate to the Entries page, check out [Part I - Exploring NOMAD](part1.md#entries_section).
+
+<!-- Screenshot of the Precision section -->
+
+Since these quantities are new and might not be commonly known, the emphasis is placed here on their definitions, as well as, the bare minimum of theoretical knowledge required to handle them.
+There are also a couple example instructions guiding you to a specific entry or for downloading processed data into a notebook.
+Also watch out for the boxes with a pencil sign.
+They delve deeper into some topics that are too tangential for the tutorial.
+
+## Reciprocal space
+
+In periodic systems, the most universal numerical parameter is the integration of the reciprocal, or k-space, and its sampling.
+With the sampling points often being spaced at fixed intervals, one can define a homogeneous _k-density_
+$= \frac{\text{no. k-points}}{||\text{k-lattice vector}||}$.
+As the k-density ramps up, the Bloch wavefunction converges. <!-- phrase better -->
+Note that each dimension has its own k-density, which do not have to coincide.
+To ensure that NOMAD users obtain data that meets their convergence needs, **k-line density** only shows the lowest value.
+
+<p align="center">
```

Comment:
There is a syntax to add zooming to the png. Example:

```
<div class="click-zoom">
    <label>
        <input type="checkbox">
        <img src="../assets/part1_explore/explore.png" alt="Explore GUI page" width="90%" title="Explore GUI page">
    </label>
</div>
```

You just need to move the img src to the corresponding part. Here and below in the others.

Reply:
I know, however, not all figures require a zoom, though. I only apply zoom to the GUI screenshots, since those are too small to see.